src/main/java/de/muenchen/dave/domain/mapper/detektor/MessstelleMapper.java

```diff
@@ -60,7 +62,9 @@ default void updateMessquerschnitt(Messquerschnitt actual, EditMessquerschnittDT
 
     @AfterMapping
     default void updateMessstelleAfterMapping(@MappingTarget Messstelle actual, EditMessstelleDTO dto) {
-        actual.setGeprueft(true);
+        if (!MessstelleStatus.IN_PLANUNG.equals(actual.getStatus())) {
+            actual.setGeprueft(true);
+        }
         // Suchworte setzen
```

Comment:
unnötiger kommentar

Reply:
Erachte ich als nicht unnötig, da das setzen der Suchworte komplexer werden kann und so markiert ist, wo dies gemacht wird

Reply:
Dann sollte das folgende eigentlich eher in eine eigene funktion (bsp. `setSuchworte()`) welche hier aufgerufen wird. 

Sehe gerade, dass das aber in dem MR gar nicht geändert wurde, daher ist das auch nicht teil davon und muss hier nicht geändert werden. Daher Resolve ich das ganze.